proto/v1/run_function.proto

```diff
@@ -16,66 +16,71 @@
 
 syntax = "proto3";
 
-//buf:lint:ignore PACKAGE_DIRECTORY_MATCH // Too late to fix this now.
+// This package defines the RPC for both composition and operation functions.
```

Comment:
refresh my memory - is the protobuf stuff just copied over from c/c repo?

Reply:
Yeah it is.